### ISSUE TITLE
Installation wizard - Updating of 'step' indication

### ISSUE THREAD

**Issue by crsfalzon** [SimplyVC/panic#196]
### User Story

As a node operator, I want to have a textual information about my progress when setting up a sub-chain, so that I am aware of the progress made, and the amount of steps that remaining for my setup to be completed.

### Description

Since the Review page will not be implemented during this iteration, the current 'step x/7' is incorrect. 

![step1of7](https://user-images.githubusercontent.com/89395527/167360804-a9be19e9-3047-46c4-b76f-a0d77f69fa0b.PNG)

### Requirements

Update the 'steps' so that we have:

- For Blockchain Setup - step 1/5
- For Channels Setup - step 2/5
- For Nodes Setup - step 3/5
- For Repositories Setup - step 4/5
- For Alerts Setup - step 5/5

### Blocked by

Not dependent on another ticket to be tackled.

### Acceptance criteria

**Scenario**: Node operator goes through installation wizard 
**Given**: The node operator is setting up sub-chains for monitoring from the installation wizard 
**When**: He configures his blockchain setup -> channels setup -> nodes setup -> repos setup ->  alerts setup
**Then**: The 'steps' information goes step 1/5 -> step 2/5 -> step 3/5 -> step 4/5 -> step 5/5
